.github/ISSUE_TEMPLATE/question.md

```diff
@@ -28,8 +28,9 @@ Documents describing various SDK features are [available here](https://github.co
 ## Step 3. Learn by example
 
 Clone the repo and search for clues in:
-- [Examples directory](https://github.com/microsoft/cpp_client_telemetry/tree/master/examples)
-- [API Functional tests](https://github.com/microsoft/cpp_client_telemetry/blob/master/tests/functests/APITest.cpp)
+
+- [Examples directory](https://github.com/microsoft/cpp_client_telemetry/tree/main/examples)
```

Comment:
Did you want to update this as well?

Reply:
Nope. This links to a folder rather than a file, relative link won't work well (e.g. if you click the link on a local machine).